modules/python/test/test_misc.py

```diff
@@ -98,13 +98,14 @@ def test_overload_resolution(msg, *args, **kwargs):
             no_exception_msg = 'Overload resolution failed without any exception for: "{}"'.format(msg)
             wrong_exception_msg = 'Overload resolution failed with wrong exception type for: "{}"'.format(msg)
             with self.assertRaises((cv.error, Exception), msg=no_exception_msg) as cm:
-                cv.utils.testOverloadResolution(*args, **kwargs)
+                res = cv.utils.testOverloadResolution(*args, **kwargs)
+                self.fail("Unexpected result for {}: '{}'".format(msg, res))
```

Comment:
`assertRaises` should mark the test as failed with a proper `no_exception_msg` if exception doesn't occur.
What is the purpose of this line?

Reply:
Dump result of the call.